tests/plugins/test_sendmail.py

```diff
@@ -213,6 +217,8 @@ class WF(object):
         assert p._should_send(rebuild, success, auto_canceled, manual_canceled) == expected
 
     @pytest.mark.parametrize('success', (True, False))
+    @pytest.mark.parametrize('annotations', (True, False))
```

Comment:
I think it would make more sense to combine `annotations` and `metadata_stored` into a single entry with an additional `expect_error` param. That way it's more explicit that certain combinations of `annotations` and `metadata_stored` values are expected to raise an error. This is preferable to adding this conditional in the test body.